zebra-state/src/service/finalized_state/disk_db.rs

```diff
@@ -512,6 +512,38 @@ impl DiskWriteBatch {
 }
 
 impl DiskDb {
+    /// Prints rocksdb metrics for each column family along with total database disk size, live data disk size and database memory size.
+    pub fn print_db_metrics(&self) {
+        let mut total_size_on_disk = 0;
+        let mut total_size_in_mem = 0;
+        let mut total_live_size_on_disk = 0;
+        let db: &Arc<DB> = &self.db;
+        let db_options = DiskDb::options(); 
+
+        let column_families = DiskDb::construct_column_families(&db_options, db.path(), &[]);
+    
+        for cf_descriptor in column_families {
+            let cf_name = &cf_descriptor.name();
+            let cf_handle = db.cf_handle(cf_name).expect("Column family handle must exist");
+    
+            let live_data_size = db.property_int_value_cf(cf_handle, "rocksdb.estimate-live-data-size").unwrap_or(Some(0));
+            let total_sst_files_size = db.property_int_value_cf(cf_handle, "rocksdb.total-sst-files-size").unwrap_or(Some(0));
+    
+            let cf_disk_size = live_data_size.unwrap_or(0) + total_sst_files_size.unwrap_or(0);
+            total_size_on_disk += cf_disk_size;
+            total_live_size_on_disk += live_data_size.unwrap_or(0);
+    
+            let mem_table_size = db.property_int_value_cf(cf_handle, "rocksdb.size-all-mem-tables").unwrap_or(Some(0));
+            total_size_in_mem += mem_table_size.unwrap_or(0);
+    
+            info!("Column Family: {}, Disk Size: {} bytes, Memory Size: {} bytes", cf_name, cf_disk_size, mem_table_size.unwrap_or(0));
```

Comment:
I think it can be maybe better to have a single line of log with all the column families and/or reduce the log level to debug. It looks a bit too much repeated information.

```
2024-03-11T23:09:18.533634Z  INFO {zebrad="1cf4e6c" net="Main"}: zebra_state::service::finalized_state::disk_db: Column Family: default, Disk Size: 0 bytes, Memory Size: 2048 bytes
2024-03-11T23:09:18.533754Z  INFO {zebrad="1cf4e6c" net="Main"}: zebra_state::service::finalized_state::disk_db: Column Family: hash_by_height, Disk Size: 36825102 bytes, Memory Size: 2048 bytes
2024-03-11T23:09:18.533797Z  INFO {zebrad="1cf4e6c" net="Main"}: zebra_state::service::finalized_state::disk_db: Column Family: height_by_hash, Disk Size: 35025601 bytes, Memory Size: 2048 bytes
2024-03-11T23:09:18.533894Z  INFO {zebrad="1cf4e6c" net="Main"}: zebra_state::service::finalized_state::disk_db: Column Family: block_header_by_height, Disk Size: 1220638770 bytes, Memory Size: 2048 bytes
2024-03-11T23:09:18.535326Z  INFO {zebrad="1cf4e6c" net="Main"}: zebra_state::service::finalized_state::disk_db: Column Family: tx_by_loc, Disk Size: 29375250200 bytes, Memory Size: 2048 bytes
2024-03-11T23:09:18.535423Z  INFO {zebrad="1cf4e6c" net="Main"}: zebra_state::service::finalized_state::disk_db: Column Family: hash_by_tx_loc, Disk Size: 359288552 bytes, Memory Size: 2048 bytes
2024-03-11T23:09:18.535489Z  INFO {zebrad="1cf4e6c" net="Main"}: zebra_state::service::finalized_state::disk_db: Column Family: tx_loc_by_hash, Disk Size: 335408928 bytes, Memory Size: 2048 bytes
2024-03-11T23:09:18.535542Z  INFO {zebrad="1cf4e6c" net="Main"}: zebra_state::service::finalized_state::disk_db: Column Family: balance_by_transparent_addr, Disk Size: 261076202 bytes, Memory Size: 2048 bytes
2024-03-11T23:09:18.535680Z  INFO {zebrad="1cf4e6c" net="Main"}: zebra_state::service::finalized_state::disk_db: Column Family: tx_loc_by_transparent_addr_loc, Disk Size: 2272709179 bytes, Memory Size: 2048 bytes
2024-03-11T23:09:18.535788Z  INFO {zebrad="1cf4e6c" net="Main"}: zebra_state::service::finalized_state::disk_db: Column Family: utxo_by_out_loc, Disk Size: 1644264579 bytes, Memory Size: 2048 bytes
2024-03-11T23:09:18.535869Z  INFO {zebrad="1cf4e6c" net="Main"}: zebra_state::service::finalized_state::disk_db: Column Family: utxo_loc_by_transparent_addr_loc, Disk Size: 647478942 bytes, Memory Size: 2048 bytes
2024-03-11T23:09:18.535914Z  INFO {zebrad="1cf4e6c" net="Main"}: zebra_state::service::finalized_state::disk_db: Column Family: sprout_nullifiers, Disk Size: 98009084 bytes, Memory Size: 2048 bytes
2024-03-11T23:09:18.535963Z  INFO {zebrad="1cf4e6c" net="Main"}: zebra_state::service::finalized_state::disk_db: Column Family: sprout_anchors, Disk Size: 197857825 bytes, Memory Size: 2048 bytes
2024-03-11T23:09:18.536008Z  INFO {zebrad="1cf4e6c" net="Main"}: zebra_state::service::finalized_state::disk_db: Column Family: sprout_note_commitment_tree, Disk Size: 9860 bytes, Memory Size: 2048 bytes
2024-03-11T23:09:18.536051Z  INFO {zebrad="1cf4e6c" net="Main"}: zebra_state::service::finalized_state::disk_db: Column Family: sapling_nullifiers, Disk Size: 0 bytes, Memory Size: 2048 bytes
2024-03-11T23:09:18.536098Z  INFO {zebrad="1cf4e6c" net="Main"}: zebra_state::service::finalized_state::disk_db: Column Family: sapling_anchors, Disk Size: 2428 bytes, Memory Size: 2048 bytes
2024-03-11T23:09:18.536136Z  INFO {zebrad="1cf4e6c" net="Main"}: zebra_state::service::finalized_state::disk_db: Column Family: sapling_note_commitment_tree, Disk Size: 2406 bytes, Memory Size: 2048 bytes
2024-03-11T23:09:18.536175Z  INFO {zebrad="1cf4e6c" net="Main"}: zebra_state::service::finalized_state::disk_db: Column Family: sapling_note_commitment_subtree, Disk Size: 0 bytes, Memory Size: 2048 bytes
2024-03-11T23:09:18.536215Z  INFO {zebrad="1cf4e6c" net="Main"}: zebra_state::service::finalized_state::disk_db: Column Family: orchard_nullifiers, Disk Size: 0 bytes, Memory Size: 2048 bytes
2024-03-11T23:09:18.536268Z  INFO {zebrad="1cf4e6c" net="Main"}: zebra_state::service::finalized_state::disk_db: Column Family: orchard_anchors, Disk Size: 2428 bytes, Memory Size: 2048 bytes
2024-03-11T23:09:18.536315Z  INFO {zebrad="1cf4e6c" net="Main"}: zebra_state::service::finalized_state::disk_db: Column Family: orchard_note_commitment_tree, Disk Size: 2406 bytes, Memory Size: 2048 bytes
2024-03-11T23:09:18.536358Z  INFO {zebrad="1cf4e6c" net="Main"}: zebra_state::service::finalized_state::disk_db: Column Family: orchard_note_commitment_subtree, Disk Size: 0 bytes, Memory Size: 2048 bytes
2024-03-11T23:09:18.536401Z  INFO {zebrad="1cf4e6c" net="Main"}: zebra_state::service::finalized_state::disk_db: Column Family: history_tree, Disk Size: 0 bytes, Memory Size: 2048 bytes
2024-03-11T23:09:18.536451Z  INFO {zebrad="1cf4e6c" net="Main"}: zebra_state::service::finalized_state::disk_db: Column Family: tip_chain_value_pool, Disk Size: 7168 bytes, Memory Size: 2048 bytes
```

Reply:
In the recent commit the column families logs are now reduced to "debug" only.